File: lib/src/main/java/br/com/sbs/android_lib_util/util/JsonUtil.java
package br.com.sbs.android_lib_util.util;

import org.json.JSONException;
import org.json.JSONObject;

/**
 * Created by mvdkpj02 on 12/26/17.
 */

public class JsonUtil {

    public static Object readKey(String json, String key){
        try {
            JSONObject jsonObject = new JSONObject(json);
            return jsonObject.has(key) ? jsonObject.opt(key) : null;
        } catch (JSONException e) {
            e.printStackTrace();
        }
        return null;
    }
}

File: lib/src/main/java/br/com/sbs/androidLibUtil/util/JsonUtil.kt
package br.com.sbs.androidLibUtil.util

import org.json.JSONException
import org.json.JSONObject

/**
 * Created by Valmir Júnior on 26/12/17.
 */

object JsonUtil {

    fun readKey(json: String, key: String): Any? {
        try {
            val jsonObject = JSONObject(json)
            return if (jsonObject.has(key)) jsonObject.opt(key) else null
        } catch (e: JSONException) {
            e.printStackTrace()
        }

        return null
    }
}
